simpleflow/swf/mapper/exceptions.py

```diff
@@ -169,13 +147,13 @@ def is_unknown_resource_raised(error, *args, **kwargs):
     :type  error: Exception
```

Comment:
👀  (Ignore me)

Reply:
I don't know if those comments are still useful now that we have proper type annotations? Or is it because it's "Exception" (== too broad)?

Reply:
Proper type annotations; but this needs a full pass on the code...